paper-server/patches/features/0003-Entity-Activation-Range-2.0.patch

```diff
@@ -242,7 +242,7 @@ index 0000000000000000000000000000000000000000..ae2bb9a73106febfe5f0d090abd4252b
 +        }
 +        // special cases.
 +        if (entity instanceof final LivingEntity living) {
-+            if (living.onClimbable() || living.jumping || living.hurtTime > 0 || !living.activeEffects.isEmpty() || living.isFreezing()) {
++            if ((living.blockPosition().equals(living.getLastClimbablePos().orElse(null)) || living.onClimbable()) || living.jumping || living.hurtTime > 0 || !living.activeEffects.isEmpty() || living.isFreezing()) {
```

Comment:
I think we can do this:
> living.getLastClimbablePos().map(living.blockPosition()::equals).orElse(false)

As if climbable pos is unset, we can assume that they are not climbing.

Reply:
I’m against using a lambda here, perhaps we should use the same code as now and just without the `||`

Reply:
then sure

Reply:
changed this 👍